### DISCUSSION TITLE
Can't upload with Author and Editor roles.

### DISCUSSION THREAD

**Discussion by rezicosta** [strapi/strapi#7753] [Legacy Topics]
Does strapi allow media library access to the Author and Editor roles?

**Comment by derrickmehaffy** [comment]
Yes the Editor role does have access to the Media Library (full control)
http://localhost:1337/admin/settings/roles/2
![image](https://user-images.githubusercontent.com/8593673/92516492-25516300-f1ca-11ea-8a4b-34b37e36bdec.png)

The Author role does have access, however access is limited to only items they upload (you can see this from the little dots next to the permissions)
http://localhost:1337/admin/settings/roles/3
![image](https://user-images.githubusercontent.com/8593673/92516583-431ec800-f1ca-11ea-8407-5be66684cd6d.png)

Here are the more advanced policies set in place for the Author role.
![image](https://user-images.githubusercontent.com/8593673/92516791-8c6f1780-f1ca-11ea-9f58-1971abab5673.png)
![image](https://user-images.githubusercontent.com/8593673/92516940-c4765a80-f1ca-11ea-94e3-8b7c1f8f326b.png)

**Comment by trilobiet** [comment]
Had the same problem here. Then I found this discussion and after seeing @derrickmehaffy answer with the pics I installed a fresh Strapi and everything works as expected. Probably messed up something when migrating settings from an older version (currently used version is 3.1.4).

Actually for a while I feared editors not being able to access uploads was some 'feature' to lead you to the paid version, but fortunately it is not (phew...) 

This really is some essential feature, makes me very happy.

**Reply by derrickmehaffy** [reply]
Awesome :)

**Reply by rezicosta** [reply]
@trilobiet Yeap, thought the same 😂. It seems that bug appears after installing something in the project. It works finely when having a fresh one.
